gangplank/ocp/cosa-pod.go

```diff
@@ -335,3 +388,159 @@ func streamPodLogs(logging *bool, pod *v1.Pod) error {
 
 	return nil
 }
+
+// encodeToJSON renders the JSON definition of a COSA pod. Podman prefers
```

Comment:
Not specifically related to this code but "COSA pod" feels ambiguous to me; how about e.g. "controller pod" and "worker pod"?